draft-ietf-dnsop-svcb-https.md

```diff
@@ -984,7 +988,7 @@ mandatory", i.e. mandatory if they are present in an RR.  The SvcParamKey
 "mandatory" is used to indicate any mandatory keys for this RR, in addition to
 any automatically mandatory keys that are present.
 
-A ServiceMode RR is considered "compatible" with a client if the client
+A ServiceMode RR is considered "compatible" by a client if the client
```

Comment:
This one I'm not entirely sure about.
It is perhaps possible that it makes sense to think about a RR being "compatible with" a client, if the RR itself is the focus of the concept.  Yet the rest of the discussion here seems focused on client processing, and the phrasing is a little weird to me anyway, so I think this is probably better.

Reply:
I can also see this going either way.

Reply:
"compatible with" seems more natural to me, but on matters of comprehensibility I defer to the reader.